terraform/development/main.tf

```diff
@@ -33,12 +33,39 @@ locals {
 module "database" {
   source = "../modules/rds"
 
-  prefix                            = local.prefix
-  allocated_storage                 = 5
-  db_subnet_group_name              = module.networking.db_private_subnet_group_name
-  ingress_source_security_group_ids = []
-  instance_class                    = "db.t3.micro"
-  vpc_id                            = module.networking.vpc_id
+  prefix                             = local.prefix
+  allocated_storage                  = 5
+  db_subnet_group_name               = module.networking.db_private_subnet_group_name
+  ingress_from_ecs_security_group_id = module.ecs.ecs_security_group_id
+  instance_class                     = "db.t3.micro"
+  vpc_id                             = module.networking.vpc_id
+}
+
+module "ecs" {
+  source = "../modules/ecs"
+
+  prefix = local.prefix
+  additional_task_role_policy_arns = {
+    "RDS_access" : module.database.rds_data_access_policy_arn
+    "Redis_access" : "arn:aws:iam::aws:policy/AmazonElastiCacheFullAccess"
+  }
+  ecr_repository_url                = "815624722760.dkr.ecr.eu-west-2.amazonaws.com/core-ecr"
+  egress_to_db_security_group_id    = module.database.rds_security_group_id
+  egress_to_redis_security_group_id = module.redis.redis_security_group_id
+  ecs_environment_variables = [
+    { Name = "API_USER", Value = "dluhc-user" },
+    { Name = "APP_HOST", Value = "" },
+    { Name = "CSV_DOWNLOAD_PAAS_INSTANCE", Value = "" },
+    { Name = "RAILS_ENV", Value = "staging" },
+    { Name = "RAILS_LOG_TO_STDOUT", Value = "true" },
+    { Name = "RAILS_SERVE_STATIC_FILES", Value = "true" }
+  ]
+  ecs_parameters         = module.parameter_store.parameter_arns
+  ecs_task_cpu           = 512
+  ecs_task_desired_count = 2
+  ecs_task_memory        = 1024

```

Comment:
In PaaS the application instances have the following specs:

Dev
- 1 instance with:
- Unknown CPU
- 1GiB RAM
- 1GiB Disk

Staging
- 2 instances each with:
- Unknown CPU
- 1GiB RAM
- 1GiB Disk

Production
- 4 instances each with:
- Unknown CPU
- Memory 1GiB
- Disk 1GiB

For now I've just configured all envs with the same but we should fine tune this down the line. Generally for staging and prod I'd expect to have at least 2 parallel tasks running